tofu/geom/_core.py

```diff
@@ -1969,7 +1969,7 @@ def from_txt(
             if pfe[-4:] == '.csv':
```

Comment:
and just because we are already changing here, would you mind putting `pfe.endswith(".csv")`? (If you liked that solution)

Reply:
Done, I agree this is cleaner